filesystem/path.h

```diff
@@ -11,7 +11,7 @@
 
 #include "fwd.h"
 #include <string>
-#include <vector>
+#include <vector>r
```

Comment:
I actually did test the code... however, I copied it into a pull request using the github web interface, thats where the typo must have slipped in ;)